FiveCalls/FiveCalls/IssuesViewController.swift

```diff
@@ -152,70 +175,93 @@ class IssuesViewController : UITableViewController {
 
-        return viewModel.issues.count
+        return viewModel.categories[section].issues.count + moreIssuesRow
     }
     
     override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
-        let notAButton = BorderedButton(frame: CGRect(x: 0, y: 0, width: tableView.frame.size.width, height: 26.0))
-        notAButton.setTitle(R.string.localizable.whatsImportantTitle(), for: .normal)
-        notAButton.setTitleColor(.fvc_darkBlueText, for: .normal)
-        notAButton.backgroundColor = .fvc_superLightGray
-        notAButton.titleLabel?.font = Appearance.instance.headerFont
-        notAButton.borderWidth = 1
-        notAButton.borderColor = .fvc_mediumGray
-        notAButton.topBorder = true
-        notAButton.bottomBorder = true
-        return notAButton
+        if shouldShowAllIssues {
+            return headerWithTitle(title: viewModel.categories[section].name)
+        }
+        return nil
     }
-    
+
     override func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
-        return 26.0
+        if shouldShowAllIssues {
+            return 26.0
+        }
+        return 0
     }
-    
+
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        guard indexPath.row < viewModel.issues.count else {
+        guard indexPath.row < viewModel.categories[indexPath.section].issues.count else {
             let cell = tableView.dequeueReusableCell(withIdentifier: R.reuseIdentifier.moreIssuesCell, for: indexPath)!
             return cell
         }
 
         let cell = tableView.dequeueReusableCell(withIdentifier: R.reuseIdentifier.issueCell, for: indexPath)!
-        let issue = viewModel.issues[indexPath.row]
+        let issue = viewModel.categories[indexPath.section].issues[indexPath.row]
+        cell.isHidden = !shouldShowAllIssues && issue.inactive
         cell.titleLabel.text = issue.name
         if let hasContacted = logs?.hasCompleted(issue: issue.id, allContacts: issue.contacts) {
             cell.checkboxView.isChecked = hasContacted
         }
         return cell
     }
 
+    override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        guard indexPath.row < viewModel.categories[indexPath.section].issues.count else {
+            return UITableViewAutomaticDimension
+        }
+        let issue = viewModel.categories[indexPath.section].issues[indexPath.row]
+        // Hide the row (height 0) if the issue for that cell is inactive and
+        // we are currently showing active issues only.
+        if !shouldShowAllIssues && issue.inactive {
+            return 0
+        }
+        return UITableViewAutomaticDimension
+    }
+
+    override func tableView(_ tableView: UITableView, estimatedHeightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 75
+    }
```

Comment:
This can be set in the storyboard (or in `viewDidLoad`) since it's just a static value and doesn't vary by section, etc...

Reply:
I am not sure if estimated height can be set from the Storyboard. I know height can be, but not sure about estimated height. Will take a look. Otherwise will just set this from viewDidLoad. Thank you.

Reply:
Estimated height can definitely be set in the storyboard.